### DISCUSSION TITLE
Can you place current line to center of the screen when using Ctrl+d and Ctrl+u.

### DISCUSSION THREAD

**Discussion by pypp** [vscode-neovim/vscode-neovim#1088] [Q&A] [answered]
Can you place current line to center of the screen when using Ctrl+d and Ctrl+u.
in  vim I would just remap Ctrl+d to be Ctrl+d zz.

**Comment by suntsu42** [comment]
I second this question. 
I do have this remap for neovim where it works,  but vscode seems to ignore it.
Or can this be done by adding a mapping in keybindings.json?

**Reply by martin-braun** [reply, marked as answer]
I will quote myself from #1234: 

> [...] I can't get the `zz` mapping to work. I told it to pass `<C-d>` and `<C-u>` to Neovim, so the scrolling is closer to the original, but it fails to scroll the window now, unless the cursor gets out of sight.
> 
> Unfortunately, `10H` isn't working either, otherwise I could scroll based on `winheight(0)/4`. Mappings `<C-d>zz` should actually be enough, but mapping `zz` isn't even working.

The hope is to recover the absolute original, but the base problem is shared. Remapping `zz` doesn't work in vscode-neovim.
